bootstrap.py

```diff
@@ -639,6 +639,20 @@ def get_api_port():
 print "This script is designed to register new systems or to migrate an existing system to a Foreman server with Katello"
 
 
+def check_prerequisite():
+    rpm_sat = ['satellite', 'satellite-capsule']
```

Comment:
I think this should be

    ['katello', 'foreman-proxy-content', 'katello-capsule']

that will work both up- and downstream, and on new (3.4) and old (3.2) releases.